src/simulations/collect.jl

```diff
@@ -53,6 +63,29 @@ two `DataFrame`s, one for agent-level data and one for model-level data.
 
 By default both keywords are `nothing`, i.e. nothing is collected/aggregated.
 
+## Mixed-Models
+
+  For mixed-models, the `adata` keyword has some additional options & properties.
+  An additional column `agent_type` will be placed in the output
+  dataframe.
+
+  In the case that data is needed for one agent type that does not exist
+  in a second agent type, `missing` values will be added to the dataframe.
+
+  **Warning:** Since this option is inherently type unstable, try to avoid this
+  in a performance critical situation. Performance will depend on specifics, but benchmarks
+  see a 10x slowdown when using `Symbol` matching and 26x for `Function` properties.
+
+  Aggregate functions will fail if `missing` values are not handled explicitly.
+  If `a1.weight` but `a2` (type: Agent2) has no `weight`, use
+  `a2(a) = a isa Agent2; adata = [(:weight, sum, a2)]` to filter out the missing results.
+
+  Data collection will fail if two agent types a property with the same name but different
+  subtypes, and are expected to aggregate with each other. For example `a1.weight::Float64`
+  and `a2.weight::Int64` with `adata = [(:weight, mean)]`. To achive this, it's best to
+  make a promote function: `fweight(a) = Float64(a.weight)`, then use
+  `adata = [(fweight, mean)]`.
+
```

Comment:
I would remove this entire thing. It is not Agents.jl related but super basic Julia knowledge.